cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -59,7 +58,8 @@ export class PassportAuth implements Auth {
    * @param sessionOpts - Session options to be used by express-session
    */
   public init(app: express.Express, sessionOpts: SessionOptions) {
-    this.log.info('Initializing express app to use express session and passport');
+    logger.info('Initializing express app to use express session and passport',
```

Comment:
I understand, but I'm referring to completely separate problem.
For the moment you have getter and direct field access. 
Having both makes no sense. 

> no need for getting
But then how you get it :)